### PR TITLE
Tests for welcome activity

### LINKED ISSUE #2644: Extend WelcomeActivityTest
**Summary:** 

Would be good to extend WelcomeActivityTest to cover more cases. These are taken from #2636:

- Test swiping the correct number of times and then clicking 'yes' exits WelcomeActivity
- ~Test that on beta clicking the 'skip tutorial' button exits WelcomeActivity~
- ~Test we can swipe both ways (i.e. backwards and forwards)~
- Test swiping beyond the proper bounds shouldn't do anything e.g. swiping left on the first page or right on the last page

### PR DESCRIPTION
**Pull request by vanshikaarora**
**Description (required)**

Fixes #2644 Extend WelcomeActivityTest

Test swiping beyond the proper bounds shouldn't do anything e.g. swiping left on the first page or right on the last page

Added the above two tests

**Tests performed (required)**

Tested ProdDebug on vivo 1713 with API level 24.

### CODE REVIEW

File: app/src/androidTest/java/fr/free/nrw/commons/WelcomeActivityTest.kt [hunk 1]
```diff
@@ -70,4 +71,35 @@ class WelcomeActivityTest {
                 .perform(ViewActions.swipeRight())
         assert(true)
     }
+
+    @Test
+    fun swipeBeyondBounds(){
+        if (!ConfigUtils.isBetaFlavour()){
```
Comment (domdomegg) [inline app/src/androidTest/java/fr/free/nrw/commons/WelcomeActivityTest.kt]: This test should probably still run on beta

File: app/src/androidTest/java/fr/free/nrw/commons/WelcomeActivityTest.kt [hunk 2]
```diff
@@ -70,4 +71,35 @@ class WelcomeActivityTest {
...
+                    onView(withId(R.id.welcomePager))
+                            .perform(ViewActions.swipeRight())
+                    assert(false)
+                }}
+        }
+    }
+
+    @Test
+    fun swipeTillLastAndFinish(){
+        if (!ConfigUtils.isBetaFlavour()){
```
Comment (domdomegg) [inline app/src/androidTest/java/fr/free/nrw/commons/WelcomeActivityTest.kt]: This test should probably still run on beta

**Comment by vanshikaarora** [comment]
Thanks for approving the PR @maskaravivek @domdomegg :)

**Comment by domdomegg** [comment]
@vanshikaarora Can you create another PR so the two tests also run on beta (i.e. remove the if condition wrapping them)

**Comment by domdomegg** [comment]
And tag me in to review it :)

**Pull request merged**